README.markdown

```diff
@@ -458,10 +550,13 @@ var names: [String] = []
 
 **NOTE**: Following this guideline means picking descriptive names is even more important than before.
 
+### Arrays
```

Comment:
I'd say collections, or `CollectionType`, since we can use it on more types than just array.